### ISSUE TITLE
Document `stdin` table name for piped input

### ISSUE THREAD

**Issue by abathur** [dinedal/textql#70]
Not a huge problem, but I had to spend a little more effort than ideal figuring out to run an update statement against the `stdin` table for a piped CSV. It would be ideal if this was documented.

Easy enough to answer with a query:
``` blah blah blah | textql -sql "SELECT name FROM sqlite_master WHERE type='table'"```

But, of course, I tried the short syntax, `pipe`, checking the readme, and searching the code thinking it might just be a random string before I thought about just querying.

**Comment by geekscrapy** [comment]
Yes, what are the default tables created for the likes of stdin?

**Comment by gabrielf** [comment]
The table name for stdin is `stdin`. Found this out by guessing :-)
